docker/index.js

```diff
@@ -2,100 +2,241 @@ const express = require('express');
 const AinJs = require('@ainblockchain/ain-js').default;
 const axios = require('axios');
 const dotenv = require('dotenv');
+const pinataSDK = require("@pinata/sdk");
+const fs = require("fs");
+const { Readable } = require("stream");
+const NodeCache = require("node-cache");
 dotenv.config();
 
 const { parsePath, formatPath } = require('./util');
 const { default: Ain } = require('@ainblockchain/ain-js');
+const { resolve } = require('dns');
 
 const app = express();
 
 const port = 80;
 const blockchainEndpoint = process.env.PROVIDER_URL;
 const chainId = process.env.NETWORK === 'mainnet' ? 1 : 0;
 const ain = new AinJs(blockchainEndpoint, chainId);
-const BOT_NAME = process.env.BOT_NAME;
 const BOT_PRIVKEY = process.env.AINIZE_INTERNAL_PRIVATE_KEY;
 const BOT_ADDRESS = AinJs.utils.toChecksumAddress(ain.wallet.add(BOT_PRIVKEY));
+const pinataApiKey = process.env.PINATA_API_KEY;
+const pinataSecretApiKey = process.env.PINATA_SECRET_API_KEY;
+const cache = new NodeCache();
 // set BOT_ADDRESS as default wallet
 ain.wallet.setDefaultAccount(BOT_ADDRESS);
 
-const SD_INPAINTING_ENDPOINT = "https://comcom-pp-diffusers-inpaint-dev.ucanuse.xyz";
-
 app.use(express.json());
 
 app.get('/', (req, res, next) => {
-    res.status(200)
-        .set('Content-Type', 'text/plain')
-        .send(`SD inpainting trigger is alive!
+	res.status(200)
+		.set('Content-Type', 'text/plain')
+		.send(`SD inpainting trigger is alive!
                input path: /apps/sf_ainft/sd_inpainting/$user_addr/$tokenId/$timestamp/input
                signed data path: /apps/sf_ainft/sd_inpainting/$user_addr/$tokenId/$timestamp/signed_data
```

Comment:
minor: This guide message should also be changed as the appPath changes

Reply:
Complete !